IrConsoleApp/Lecture5/Company.cs

```diff
@@ -0,0 +1,70 @@
+﻿using ClassLibraryHW5;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IrynaZharnasek.Lecture5
+{
+    public class Company
+    {
+        private string companyName;
+        private Employee[] arrayOfEmployees = new Employee[100];
+
+        public void AddNewEmployee(Employee employee)
+        {
+            for (var i = 0; i < arrayOfEmployees.Length; i++)
+            {
+                if (arrayOfEmployees[i] == null)
+                {
+                    arrayOfEmployees[i] = employee;
+                    break;
+                }
+            }
+        }
+        public void ShowTotalNumbersOfEmployees ()
+        {
+            var count =0;
+            for (var i = 0; i < arrayOfEmployees.Length; i++)
+            {
+                if (arrayOfEmployees[i] != null)
+                {
+                    count = i + 1;
+                    Console.WriteLine($"General count of employees:{count}");
+                }
+             }
+        }
+        public void ShowAllEmployees()
```

Comment:
You may create similar method in Employee class that will output in console employee's name and surname or use method Print() from Employee class and just call this method in Company class for all employees

Reply:
Done